docs/index.md

```diff
@@ -8,22 +8,28 @@
 :end-before: (DO-NOT-REMOVE-docs-intro-END)
 ```
 
```

Comment:
Keep the two-line separator. I intentionally separated ToC from the prose with bigger spacing.
```suggestion


```